src/fs/driver/nfs/nfs.c

```diff
@@ -551,6 +551,12 @@ static int nfs_create_dir_entry(node_t *parent_node) {
 			point += sizeof(vf);
 			predesc = (readdir_desc_t *) point;
 
+			if(predesc->file_attr.type == 1) {
+				predesc->file_attr.mode = S_IFREG | predesc->file_attr.mode;
```

Comment:
`predesc->file_attr.mode |= S_IFREG` will look more readable (because of long left part of the expression)

Reply:
fixed